### ISSUE TITLE
https://pkg.cfssl.org/ is missing releases

### ISSUE THREAD

**Issue by the-maldridge** [cloudflare/cfssl#995]
The last release on the site seems to be 1.2, whereas the last release of the source is significantly later.  Perhaps the hosted binaries can be updated?

**Comment by itnikita** [comment]
I seem to meet the same problem and where can I download the binary cfssl of version 1.3.3

**Comment by githubixx** [comment]
I'm maintaining a Ansible CFSSL role (https://github.com/githubixx/ansible-role-cfssl). I can't currently update the role to the latest version because of missing binaries at https://pkg.cfssl.org/. Any chance to have at least the latest binary (currently 1.3.4) there?

**Comment by bryfry** [comment]
Another example: 

https://github.com/kelseyhightower/kubernetes-the-hard-way/blob/master/docs/02-client-tools.md

That project used to link to the official releases on pkg.cfssl.org, but now it self hosts them.  Please fix this!

**Comment by cbroglie** [comment]
Thanks for your patience, we've updated our tooling and are now publishing binaries along with releases, beginning with 1.4.0 which I've just released: https://github.com/cloudflare/cfssl/releases/tag/v1.4.0.

We're also working on provided .deb and .rpm packages via http://pkg.cloudflare.com, and will publish Docker images as well.

The https://pkg.cfssl.org page will be replaced with links to all of the above shortly.
